apps/examples/webserver_test/webserver_test_main.c

```diff
@@ -530,6 +530,9 @@ static int ws_test_base(void *arg)
 	send_len = sndlen;
 	for (int i = 0; i < param->keep_alive_count; i++) {
 		read_finish = false;
+		memset(&mlen, 0, sizeof(mlen));
+		state = HTTP_REQUEST_HEADER;
```

Comment:
state should be initialized here? if it should be initialized before parse_message called, (line 601), I think it should be moved there.

Reply:
This can not be initialized before parse as state changes inside while loop.

Reply:
state never used unitl line 601, some of SQE tool (c.f svace) will treat it as a defect.

Reply:
state gets changed inside the parse_message() function multiple times and that is reason, state can not be modified before parse_message() function otherwise it will break the functionality.